test/common/json/json_fuzz_test.cc

```diff
@@ -8,10 +8,18 @@
 namespace Envoy {
 namespace Fuzz {
 
+static const size_t MaxInputSize = 64 * 1024;
+
 // We have multiple third party JSON parsers in Envoy, nlohmann/JSON, RapidJSON and Protobuf.
 // We fuzz nlohmann/JSON and protobuf and compare their results, since RapidJSON is deprecated and
 // has known limitations. See https://github.com/envoyproxy/envoy/issues/4705.
 DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  // protect against overly large JSON files
+  if (len > kMaxInputSize) {
```

Comment:
```suggestion
  if (len > MaxInputSize) {
```
(or change the name in line 11).

Reply:
Sorry, in a previous project all constant were prefix with a `k`. I don't get this out of my head. Nevertheless, how did this compile anyhow? Doesn't matter, fixed now.